requirements.in

```diff
@@ -14,3 +14,4 @@
 
 mkdocs
 mkdocs-material
+certifi>=2022.12.07
```

Comment:
Instead of semi-pinning, you can also update the requirements to latest with `--upgrade` or something. It's been a while since I've used that rule.

Reply:
Thanks!  I hadn't considered that we'd be adding a transitive dependency to our `requirements.in`.

I added the `--upgrade` version of the command to the usage comment in `requirements_lock.txt`.

Reply:
Err... rather, I moved the comment to `requirements.in`, so as not to edit an auto-generated file.  :facepalm: